bears/general/InvalidLinkBear.py

```diff
@@ -8,24 +8,25 @@
 
 
 class InvalidLinkBear(LocalBear):
-
+    DEFAULT_TIMEOUT = 2
     @classmethod
     def check_prerequisites(cls):
-        code = InvalidLinkBear.get_status_code_or_error("http://www.google.com")
+        code = InvalidLinkBear.get_status_code_or_error(
+            "http://www.google.com", InvalidLinkBear.DEFAULT_TIMEOUT)
```

Comment:
`InvalidLinkBear.DEFAULT_TIMEOUT` -> `cls.DEFAULT_TIMEOUT`
that's what the `cls` parameter is for :)
oh the same works also with `get_status_code_or_error` :+1: